core/templates/layout.user.php

```diff
@@ -87,7 +87,10 @@
 
 		<main id="content" class="app-<?php p($_['appid']) ?>">
 			<h1 class="hidden-visually" id="page-heading-level-1">
-				<?php p(!empty($_['pageTitle'])?$_['pageTitle']:$theme->getName()); ?>
+				<?php p((!empty($_['application']) && !empty($_['pageTitle'] && $_['application'] != $_['pageTitle'])
+					? $_['application'].': '.$_['pageTitle']
+					: (!empty($_['pageTitle']) ? $_['pageTitle']
+					: $theme->getName())));	?>
```

Comment:
A typo in brackets (second empty is not closed):
 
```suggestion
				<?php 
					p((!empty($_['application']) && !empty($_['pageTitle']) && $_['application'] != $_['pageTitle'])
						? $_['application'].': '.$_['pageTitle']
						: (!empty($_['pageTitle']) ? $_['pageTitle'] : $theme->getName())
					);
				?>
```

Reply:
corrected, thanks a lot!